packages/terra-menu/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 6.23.0 - (January 7, 2020)
 ------------------
 ### Added
+* Added new `headertTitle` prop for menus which have drill-in submenus
```

Comment:
whoops, looks like this got moved under a released section on merge

Reply:
oops my bad. Updated here 4b0525184b23469cb9fe417ee47bbff1a30e034b